packages/garbo/src/tasks/dailyItems.ts

```diff
@@ -138,6 +140,62 @@ function pickCargoPocket(): void {
   }
 }
 
+function bestDevilerCandy(): Item {
+  // These are replenishable notrade nodiscard items that we would prefer to use
+  const priorityUntradeableNoDiscardList =
+    $items`sugar shotgun, sugar shillelagh, sugar shank, sugar chapeau, sugar shorts, sugar shield, sugar shirt`.filter(
+      (i) => itemAmount(i) > 1,
```

Comment:
is there a reason we aren't just using `have` here?

Reply:
we can use `have(i, 2)` but I cop to not having noticed that it's > 1 instead of > 0, this is good